install/actualbudget-install.sh

```diff
@@ -14,11 +14,13 @@ network_check
 update_os
 
 msg_info "Installing Dependencies"
-$STD apt-get install -y curl
-$STD apt-get install -y sudo
-$STD apt-get install -y mc
-$STD apt-get install -y gpg
-$STD apt-get install -y git
+$STD apt-get install -y \
+  curl \
+  sudo \
+  mc \
+  gpg \
+  git \
+  build-essential
```

Comment:
Can do multiple apt-get installs if that's preferred. Just moved it to a single command as most of the install scripts I looked at did the same.

```suggestion
$STD apt-get install -y curl
$STD apt-get install -y sudo
$STD apt-get install -y mc
$STD apt-get install -y gpg
$STD apt-get install -y git
$STD apt-get install -y build-essential
```

Reply:
Thats the old method. The new is correct in the pr :smile: